docs/DeveloperGuide.md

```diff
@@ -216,33 +216,47 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 #### Implementation
 
-The sorting feature consists of two commands, `SortExpiryCommand` and `SortPriorityCommand` which extend `Command`.
+The sorting feature consists of three commands, `SortDescCommand`, `SortExpiryCommand` and `SortPriorityCommand` which extend `Command`.
 
 The sorting order is in accordance to what is likely the most useful order for the user.
 
-Thus, `SortExpiryCommand` sorts the list of food displayed by expiry date from oldest to newest, followed by priority from `HIGH` to `LOW`, followed by the lexicographical order.
+Thus, `SortDescCommand` sorts the list of food displayed by description, then by expiry date from oldest to newest, followed by priority from `HIGH` to `LOW`.
 
-Similarly, `SortPriorityCommand` sorts the list of food displayed by priority from `HIGH` to `LOW`, followed by expiry date from oldest to newest, followed by the lexicographical order.
+Similarly, `SortExpiryCommand` sorts the list of food displayed by expiry date from oldest to newest, followed by priority from `HIGH` to `LOW`, followed by description.
 
-When the commands are executed by calling `SortExpiryCommand#execute(Model model)` or `SortPriorityCommand#execute(Model model)`, the `SortedList<Food>` attribute in `model` is sorted.
+Similarly, `SortPriorityCommand` sorts the list of food displayed by priority from `HIGH` to `LOW`, followed by expiry date from oldest to newest, followed by description.
 
-This is done so by calling `model#updateSortedFoodList(Comparator<Food> comparator)` method in `model` with the relevant `Comparator<Food>` for sorting.
+When the commands are executed by calling `SortDescCommand#execute(Model model)` or `SortExpiryCommand#execute(Model model)` or `SortPriorityCommand#execute(Model model)`, the `versionedFoodInventory` attribute in `model` is sorted.
 
-Sorting of the `SortedList<Food>` attribute in `model` is reflected in the GUI when `MainWindow` calls `logic#getFilteredFoodList()`.
+This is done so by calling `model#sortFoodInventory(Comparator<Food>... comparators)` method in `model` with the relevant `comparators` for sorting.
+
+`model#setSortingComparators` and `userPref#setSortingComparatorsDescription` are then called to save the sorting information.
```

Comment:
Should we have brackets after the method names, for consistency? e.g. `model#setSortingComparators(Comparator<Food>[] sortingComparators)`

Reply:
Will do, thanks!